mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -2129,33 +2129,35 @@ func ResourceContainerCluster() *schema.Resource {
 				Type:        schema.TypeList,
 				Optional:    true,
 				MaxItems:    1,
-				DiffSuppressFunc: suppressDiffForAutopilot,
```

Comment:
Ah, this ends up being a breaking change in practice. Autopilot clusters that don't set a value at all get this back in the response:

```
  "dnsConfig": {
   "clusterDns": "CLOUD_DNS",
   "clusterDnsScope": "CLUSTER_SCOPE",
   "clusterDnsDomain": "cluster.local"
  },
```

Terraform will see a diff on that by removing the DSF on the top-level field, even though the subfields will be supressed. If there was not a default value for the block this would be safe.